app/src/main/java/com/android/sample/ui/recipeOverview/RecipeOverview.kt

```diff
@@ -74,37 +83,46 @@ import com.android.sample.resources.C.Dimension.RecipeOverview.OVERVIEW_RECIPE_R
 import com.android.sample.resources.C.Dimension.RecipeOverview.OVERVIEW_RECIPE_STAR_SIZE
 import com.android.sample.resources.C.Dimension.RecipeOverview.OVERVIEW_TIME_DISPLAY_RATE
 import com.android.sample.resources.C.Tag.PADDING
+import com.android.sample.resources.C.Tag.PRODUCT_FRONT_IMAGE_THUMBNAIL_URL
 import com.android.sample.resources.C.Tag.SMALL_PADDING
-import com.android.sample.resources.C.Tag.SwipePage.RATE_VALUE
 import com.android.sample.resources.C.TestTag.RecipeOverview.COOK_TIME_TEXT
 import com.android.sample.resources.C.TestTag.RecipeOverview.DRAGGABLE_ITEM
 import com.android.sample.resources.C.TestTag.RecipeOverview.INGREDIENTS_VIEW
 import com.android.sample.resources.C.TestTag.RecipeOverview.INGREDIENT_CHECKBOX
 import com.android.sample.resources.C.TestTag.RecipeOverview.INGREDIENT_PREFIX
 import com.android.sample.resources.C.TestTag.RecipeOverview.INSTRUCTIONS_VIEW
 import com.android.sample.resources.C.TestTag.RecipeOverview.PREP_TIME_TEXT
-import com.android.sample.resources.C.TestTag.RecipeOverview.RATING_ICON
+import com.android.sample.resources.C.TestTag.RecipeOverview.RECIPE_FRIDGE_INGREDIENTS
+import com.android.sample.resources.C.TestTag.RecipeOverview.RECIPE_FRIDGE_INGREDIENTS_TEXT
 import com.android.sample.resources.C.TestTag.RecipeOverview.RECIPE_IMAGE
-import com.android.sample.resources.C.TestTag.RecipeOverview.RECIPE_RATE
-import com.android.sample.resources.C.TestTag.RecipeOverview.RECIPE_STAR
 import com.android.sample.resources.C.TestTag.RecipeOverview.RECIPE_TITLE
 import com.android.sample.resources.C.TestTag.RecipeOverview.SLIDING_BUTTON_INGREDIENTS
 import com.android.sample.resources.C.TestTag.RecipeOverview.SLIDING_BUTTON_INSTRUCTIONS
 import com.android.sample.resources.C.TestTag.RecipeOverview.TOTAL_TIME_TEXT
 import com.android.sample.resources.C.Values.RecipeOverview.INITIAL_NUMBER_PERSON_PER_RECIPE
 import com.android.sample.ui.navigation.NavigationActions
-import com.android.sample.ui.theme.starColor
+import com.android.sample.ui.theme.graySlate
 import com.android.sample.ui.utils.Counter
 import com.android.sample.ui.utils.PlateSwipeScaffold
 import com.android.sample.ui.utils.Tag
 
 @Composable
 fun RecipeOverview(
     navigationActions: NavigationActions,
-    recipeOverviewViewModel: RecipeOverviewViewModel
+    recipeOverviewViewModel: RecipeOverviewViewModel,
+    userViewModel: UserViewModel
 ) {
 
   val currentRecipe by recipeOverviewViewModel.currentRecipe.collectAsState()
+  val fridgeIngredientsMap =
+      if (currentRecipe != null) {
+        userViewModel.mapFridgeIngredientsToCategories(
+            currentRecipe!!.ingredientsAndMeasurements.map { it.first })
+      } else {
+        null
+      }
+
+  Log.d("dsad", "RecipeDescription: $fridgeIngredientsMap , $currentRecipe")
```

Comment:
Great work on this file! 🎉 Just a quick note there’s a `Log.d` debug statement that seems to have missed its cue to leave. It’s like the last guest at the party who didn’t realize it’s cleanup time. 😄 Removing it will keep the code neat and polished. Awesome job otherwise! 🚀